frontend/eda/access/teams/components/steps/EdaSelectTeamsStep.tsx

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ITableColumn, TextCell } from '../../../../../../framework';
+// import { useTeamsFilters } from '../../hooks/useTeamsFilters';
+import { edaAPI } from '../../../../common/eda-utils';
+import { useMultiSelectListView } from '../../../../common/useMultiSelectListView';
+import { PageMultiSelectList } from '../../../../../../framework/PageTable/PageMultiSelectList';
+import { EdaTeam } from '../../../../interfaces/EdaTeam';
+import { Title } from '@patternfly/react-core';
+import styled from 'styled-components';
+
+const StyledTitle = styled(Title)`
```

Comment:
I think we should avoid styling this.
If you put the elements in a <TextContent> PF then applies the right styles to the elements.